### ISSUE TITLE
Endpoint to send data over SSL

### ISSUE THREAD

**Issue by irazasyed** [theiconic/php-ga-measurement-protocol#3]
Hi,

First of all, Great package. Just started playing around with it and it's fantastic. Was thinking of creating a wrapper for Laravel framework. :+1: 

Second, The endpoint for sending data over SSL is actually `https://ssl.google-analytics.com/collect` as per the documentation, however looking at your code it seems like you're simply changing the URI scheme (It works that way too). Does it make any difference? Shouldn't it be as per docs?

Let me know!

Thanks.

**Comment by jorgeborges** [comment]
Hi there @irazasyed

Thanks for praising the package, we put considerable effort into it. Let me know if you create that wrapper to link it from the README page.

Regarding the SSL endpoint, I guess I overlooked the "ssl." in the URL at first, and since it was working with only setting the scheme as HTTPS, I really didn't notice.

I just did a minor release **v1.1.1** where I fix this. If you installed using `"theiconic/php-ga-measurement-protocol": "~1.1"` you only need to run composer update. Thanks again for the heads up. :+1:
